docs/calculations.mdx

```diff
@@ -55,12 +55,14 @@ import ChannelMappingDefaults from "/snippets/defaults_channel_mapping.mdx";
 | ------------------------ | -------------------------------------------------------------------------------------------- |
 | `rendered_width_pixels`  | Width of the creative when rendered. Leave blank for responsive (will assume screen width)   |
 | `rendered_height_pixels` | Height of the creative when rendered. Leave blank for responsive (will assume screen height) |
-| `image_sizes`            | Array of image dimensions (eg `["300x250", "70x70"]`)                                        |
+| `image_sizes_pixels`     | Array of image dimensions (eg `["300x250", "70x70"]`)                                        |
 | `audio_duration_seconds` | Audio duration (if applicable)                                                               |
 | `video_duration_seconds` | Video duration (if applicable)                                                               |
 | `video_player`           | Video player used to render video (if applicable)                                            |
 | `other_assets_bytes`     | Metadata, text, html, etc                                                                    |
 | `ad_platforms`           | Ad serving, verification, and measurement ad platforms wrapping or embedded in the ad format |
+| `identifier`             | The identifying information, including name, channel, and device type                        |
```

Comment:
We call this `adFormat` in the docs and throughout the Scope3 application for integrate reporting. Should we be consistent here as well?

Reply:
making changes as noted here.
if `adFormat` is the correct field for consistency, then let's use that